components/accordion/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@spectrum-css/accordion",
+  "version": "2.0.0-alpha.5",
+  "description": "The Spectrum CSS accordion component",
+  "author": "",
+  "license": "Apache-2.0",
+  "main": "dist/index-vars.css",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/spectrum-css.git"
+  },
+  "bugs": {
+    "url": "https://github.com/adobe/spectrum-css/issues"
+  },
+  "scripts": {
+    "build": "gulp"
+  },
+  "peerDependencies": {
+    "@spectrum-css/icon": "^2.0.0-alpha.5",
```

Comment:
I think you should be able to put these into deps instead of peer and dev?
that way they are automatically resolved and installed and end users don't have to specify ALL of them? also, they'd get an actual error if they used the wrong version of something as opposed to just a peer warning?

we're currently set up like that, though maybe we should revisit too depending on this conversation

@devongovett I think you started us on our current dep pattern, thoughts?

Reply:
Yeah, we considered that. We were struggling with the fact that they're not real dependencies (i.e. no code in the published packages actually relies on these dependencies to execute)...

Getting an error vs a warning sounds promising, we should revisit this.